data/migrations/sql.jinja2

```diff
@@ -146,8 +146,12 @@ DECLARE
   row ALIAS FOR $1;
 BEGIN
   RETURN mz_calculate_min_zoom_pois_(
-{%- for param in layers['pois']['params'] %}
+{%- for param in layers['pois']['params'] -%}
+{%- if param.key == '"ele"' %}
+        row.tags->'ele',
+{%- else %}
```

Comment:
Yes, for `planet_osm_polygon`. However, `planet_osm_point`, at least on my local DB, has it as a "full" column. This seemed like the simplest (although kinda klidgy) way to avoid having to introduce a distinction between the two tables which are otherwise very similar.